src/utils/vc/crypto/common/DockCryptoKeyPair.js

```diff
@@ -37,6 +37,10 @@ export default withExtendedStaticProperties(
       return new this(options);
     }
 
+    static paramGenerator() {
```

Comment:
Maybe would be better to define this function as
```
class DockCryptoKeyPair {
   ...
   static get generateParams() { return this.SignatureParams.getSigParamsOfRequiredSize; }
   ...
}
```

Reply:
Yes, this is cleaner. Fixed.